api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java

```diff
@@ -35,6 +42,31 @@ public String readMandrillApiKey() {
     return mandrillKeys.getString("api-key");
   }
 
+  public void copyAllDemoNotebooks(String workspaceBucket)  {
+    Storage storage = StorageOptions.getDefaultInstance().getService();
+    Bucket demoBucket = storage.get(getDemosBucketName());
+    StreamSupport
+        .stream(demoBucket.list().getValues().spliterator(), false)
+        .filter(blob -> blob.getBlobId().getName().endsWith(".ipynb"))
+        .map(blob -> ImmutablePair
+            .of(blob.getBlobId(), BlobId.of(workspaceBucket, "notebooks/" + blob.getBlobId().getName())))
+        .forEach(pair -> copyBlob(pair.getLeft(), pair.getRight()));
+  }
+
+  public List<JSONObject> readAllDemoCohorts() {
+    Storage storage = StorageOptions.getDefaultInstance().getService();
+    Bucket demoBucket = storage.get(getDemosBucketName());
+
+    List<JSONObject> cohortFiles = StreamSupport
+        .stream(demoBucket.list().getValues().spliterator(), false)
+        .filter(blob -> blob.getBlobId().getName().endsWith(".json"))
+        .map(blob -> new JSONObject(new String(blob.getContent()).trim()))
+        .filter(jsonObject -> jsonObject.getString("type").equalsIgnoreCase("cohort"))
+        .map(jsonObject -> jsonObject.getJSONObject("cohort"))
+        .collect(Collectors.toList());
+    return cohortFiles;
```

Comment:
nit: you could return the result of the stream instead of assigning to `cohortFiles`.